identity/aziot-hub-client-async/src/lib.rs

```diff
@@ -275,6 +276,7 @@ impl Client {
         let body = hyper::body::to_bytes(body)
             .await
             .map_err(|err| std::io::Error::new(std::io::ErrorKind::Other, err))?;
+        log::debug!("IoTHub response body {:?}", body);
```

Comment:
Remove this.

Reply:
I added this to mirror https://github.com/Azure/iot-identity-service/blob/main/identity/aziot-dps-client-async/src/lib.rs#L309.

It proved to be reasonably useful when debugging, so I'd like to keep it in. Alternatively, if you want it gone, we should get rid of the other one as well.

Reply:
@massand I vaguely remember you saying we shouldn't log cloud responses that had secrets. Is that correct?

Reply:
We shouldn't log any responses from IoT Hub due to credential information disclosure issues. DPS doesn't log any keys in its responses for 2 APIs. Maybe adding a comment in DPS client about the responses being clean would help avoid copying that behavior next time.